src/main/resources/i18n/messages.properties

```diff
@@ -514,7 +511,32 @@ UserDetailsPage.form.button.submit=Update User
 UserDetailsPage.createdDate=Created on {0}
 UserDetailsPage.modifiedDate=Modified on {0}
 UserDetailsPage.lastLogin=Last login on {0}
-UserDetailsPage.loading.message=Loading user details.
+
+UserSecurityPage.title=Security
+UserSecurityPage.alert.title=Missing Email Configuration
+UserSecurityPage.alert.description=Password reset is disabled due to this instance of IRIDA not having a configured mail server.
+
+UserChangePasswordForm.title=Change Password
+UserChangePasswordForm.alert.title=Password Policy
+UserChangePasswordForm.alert.description=Please choose a strong password that meets the following criteria:
+UserChangePasswordForm.alert.rule1=Must not be empty.
+UserChangePasswordForm.alert.rule2=Must be at least 8 characters long.
+UserChangePasswordForm.alert.rule3=Must contain at least 1 uppercase letter. (eg. 'A')
```

Comment:
Sorry when I meant add an example for the special characters rule that was the only one that needed an example. The uppercase, lowercase, and number don't require them

Reply:
Please see 0a4765a7e95993de49d21b886529fed906f4b2aa.